modules/@apostrophecms/doc/index.js

```diff
@@ -1025,6 +1025,16 @@ module.exports = {
       },
       isDraft(doc) {
         return doc.aposLocale.endsWith(':draft');
+      },
+      // Given a type name, normalize for any backwards compatibility
+      // provisions such as accepting @apostrophecms/page for
+      // @apostrophecms/any-page-type
+      normalizeType(type) {
```

Comment:
Now that we have reached beta, we should not introduce bc breaks without a compelling need.